src/store/mutations.ts

```diff
@@ -274,11 +271,13 @@ export default {
   },
 
   updateVitalInvested(state: State, payload: any) {
-    state.build.character.vitals[payload.name].invested = Number(payload.value);
+    state.build.character.vitals[payload.name].invested = Math.min(Number(payload.value), 196);
```

Comment:
Could you reference or create a constant for 196 here (something like MAX_VITAL_INVESTED) if one doesn't exist?

Reply:
You got it. 👍 https://github.com/amoeba/accharplanner/pull/398/commits/89a31b8729cf644eaf022bdcdbefbd5c091bf318